### PR TITLE
Fix (zeb-991): modals

### PR DESCRIPTION
**Pull request by danymill**

### CODE REVIEW

File: client/app/app.module.js [hunk 1]
```diff
@@ -1101,9 +1101,124 @@ const ngModule = angular
...
             }
             return $delegate;
+        });
+        $provide.decorator('$mdDateRangePicker', ($delegate, $mdDialog) => {
+            'ngInject';
+
+            const delegate = new mdDateRangePickerDelegate($delegate, $mdDialog);
+
+            return delegate.decorate();
         })
```
Comment (DmitryMishin) [inline client/app/app.module.js]: semicolon

File: client/app/app.module.js [hunk 2]
```diff
@@ -1101,9 +1101,124 @@ const ngModule = angular
...
+                    '<md-dialog-actions layout="row" layout-align="end center">',
+                    '<md-button ng-if="!config.autoConfirm" class="md-raised" ng-click="clear()">{{getLocalizationVal("Clear")}}</md-button>',
+                    '<md-button ng-if="!config.autoConfirm" class="md-raised md-primary" ng-click="ok()">{{getLocalizationVal("Ok")}}</md-button>',
+                    '</md-dialog-actions>',
+                    '</md-dialog>'].join(''),
+                parent: angular.element(document.body),
+                multiple: config.multiple,
+                targetEvent: config.targetEvent || document.body,
+                clickOutsideToClose: true,
+                fullscreen: config.model.fullscreen
```
Comment (DmitryMishin) [inline client/app/app.module.js]: comma

**Review by dzmitrymishyn — CHANGES_REQUESTED** [review]

File: client/app/app.module.js (hunk 1 above)
Comment (danymill) [reply client/app/app.module.js]: fixed

File: client/app/app.module.js (hunk 2 above)
Comment (danymill) [reply client/app/app.module.js]: fixed

**Review by dzmitrymishyn — APPROVED** [review]

**Pull request merged**